examples/cms-agilitycms/README.md

```diff
@@ -0,0 +1,326 @@
+# A statically generated blog example using Next.js and Agility CMS
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Agility CMS](https://www.agilitycms.com) as the data source.
+
+> `IMPORTANT` - This example uses Agility CMS's [**Page Management**](https://agilitycms.com/resources/posts/page-management-in-agility-cms-vs-other-headless-cmss) features. This means that the CMS ultimately drives what pages are available and what content is on each page. This enables **Editors** to focus on managing their pages, while allowing you, (the **Developer**) to focus on building UI components for the editors to compose their pages.
+
+## Demo
+
+[`LIVE` - https://next-blog-agilitycms.now.sh/](https://next-blog-agilitycms.now.sh/)
+
+[`PREVIEW MODE` - https://next-blog-agilitycms.now.sh/?agilitypreviewkey](https://next-blog-agilitycms.now.sh/?agilitypreviewkey=GzL%2fio1pLkfKc9BR1%2fC1cDQeKjL0AkwrTAJ22q3UEjAcOhyrqZejDkDv4kMlBKqrEuQxsuRyiP%2bUaykDYlJ%2fJg%3d%3d)
+
+### Related examples
+
+- [Agility CMS Sample Starter](https://github.com/agility/agilitycms-next-starter-ssg)
+- [Blog Starter](/examples/blog-starter)
+- [Sanity](/examples/cms-sanity)
+- [TakeShape](/examples/cms-takeshape)
+- [Prismic](/examples/cms-prismic)
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npm init next-app --example cms-agilitycms cms-agilitycms-app
+# or
+yarn create next-app --example cms-agilitycms cms-agilitycms-app
+```
+
+### Download manually
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/zeit/next.js/tar.gz/canary | tar -xz --strip=2 next.js-canary/examples/cms-agilitycms
+cd cms-agilitycms
+```
+
+## Configuration
+
+### How is this Different from Other CMS Examples?
+
+We believe **Editors** should have full control of their pages and what content is on each page without getting into code.
```

Comment:
I think we want to avoid using "We" because it might suggest that "We" = "Next.js team", which is not the case.

```suggestion
The key principle behind Agility CMS is that **Editors** should have full control of their pages and what content is on each page without getting into code.
```